emailvalidation/emailvalidation.go

```diff
@@ -0,0 +1,25 @@
+package emailvalidation
+
+import (
+	"net"
+	"regexp"
+	"strings"
+)
+
+var emailRegex = regexp.MustCompile("^[a-zA-Z0-9.!#$%&'*+\\/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*$")
```

Comment:
Consider adding some GoDoc here to outline what the pattern is looking for - it's helpful to others in future and also reminds you what this is doing 6months down the line when you've totally forgetting 😂

Reply:
Documentation has been added to explain the regex. Thank you @mike-kelly-methods

Reply:
👍